### PR TITLE
Fixes unit tests by providing expected argument type to close Tessels

### PR DESCRIPTION
**Pull request by johnnyman727**
@rwaldron can I get a review?

### CODE REVIEW

**Comment by rwaldron** [comment]
```
$ grunt
Running "jshint:all" (jshint) task
>> 42 files lint free.

Running "jscs:all" (jscs) task
>> 42 files without code style errors.

Running "jsbeautifier:all" (jsbeautifier) task
Beautified 42 files, changed 0 files...OK

Running "nodeunit:tests" (nodeunit) task
Segmentation fault: 11
rwaldron at nova in ~/clonez/t2-cli on jon-fix-tests
$ grunt nodeunit
Running "nodeunit:tests" (nodeunit) task
Segmentation fault: 11
```

Yikes!

**Comment by rwaldron** [comment]
JUST KIDDING.

Completely forgot that I had an older version of iojs symlinked to node.

**Comment by rwaldron** [comment]
LGTM

As a follow up, we should get some tests written that ensure: 

1. `controller.closeTesselConnections` is called when expected to be called. 
2. `controller.closeTesselConnections` is called only with an array.

**Comment by johnnyman727** [comment]
@rwaldron I have added tests as you described to all existing unit tests that should close Tessel connections afterward. Merge if it looks good!

**Comment by johnnyman727** [comment]
Actually just going to merge for now because all the tests work, @rwaldron already signed off on the actual application code change, and I'd like to have this fix for my other debugging to make sure the underlying bug isn't causing secondary issues.

File: test/unit/controller.js [hunk 1]
```diff
@@ -171,6 +171,7 @@ exports['Tessel.list'] = {
         test.equal(this.runHeuristics.callCount, 0);
         test.equal(this.closeTesselConnections.callCount, 1);
         test.equal(a.close.callCount, 1);
+        test.equal(Array.isArray(this.closeTesselConnections.args[0]), true);
```
Comment (rwaldron) [inline test/unit/controller.js]: Nice

**Pull request merged**